DEVELOPING_TIPS.md

```diff
@@ -0,0 +1,52 @@
+# Cycloid Developer Tips
+
+This file gives some tips to how to test, change or upgrade cli, for cycloid developers.
+
+The cli pipelin is available in the [cycloid-stacks](https://github.com/cycloidio/cycloid-stacks/tree/stacks/cycloid-cli) and the prod and staging cli can be checked [here](https://console.cycloid.io/organizations/cycloid/projects/cycloid-cli).
```

Comment:
I would maybe add a note to let people know this is internal notes. As those pipelines currently are in private git repos

Reply:
```suggestion
The cli pipeline is available in the [cycloid-stacks](https://github.com/cycloidio/cycloid-stacks/tree/stacks/cycloid-cli) and the prod and staging cli can be checked [here](https://console.cycloid.io/organizations/cycloid/projects/cycloid-cli).
```